make/autoconf/toolchain.m4

```diff
@@ -40,6 +40,7 @@ VALID_TOOLCHAINS_all="gcc clang microsoft"
 # These toolchains are valid on different platforms
 VALID_TOOLCHAINS_linux="gcc clang"
 VALID_TOOLCHAINS_macosx="clang"
+VALID_TOOLCHAINS_bsd="gcc clang"
```

Comment:
Note that the order here is relevant: the first toolchain mentioned will be the default. (I was under the impression that clang was more popular than gcc on the BSDs, but I might be wrong.)

Reply:
Thanks, and correct. Clang is the native compiler for at least FreeBSD.